pages/documentation/settings.md

```diff
@@ -19,20 +21,47 @@ subnav:
   href: '#utilities-settings'
 ---
 
-A good way to get started with USWDS settings is to copy the `dist/theme` directory out of the `uswds` package and into the same directory where you keep your project Sass files. Then use the `@forward` directive in Sass to include these settings **before including the main USWDS Sass**. You can use the `styles.scss` example file in the `dist/theme` directory as a starting point for your own Sass entry point:
+## Introducing USWDS settings
+USWDS is built using a suite of customizable settings that allows you to configure the Design System to fit your project's needs. These settings are Sass variables that begin with the `$theme-` prefix and are typically defined with [USWDS design tokens]({{ site.baseurl }}/design-tokens). You can find the full list of USWDS settings and their default values in the [USWDS settings tables](#general-settings).
+
+## Configuring custom USWDS settings
+With the introduction of [Sass modules](https://sass-lang.com/blog/the-module-system-is-launched) in USWDS 3, you can now create a custom configuration of USWDS by loading a single [`@use...with()`](https://sass-lang.com/documentation/at-rules/use#configuration) rule into your [Sass entry point]({{ site.baseurl }}/documentation/getting-started/developers/phase-two-compile/#step-1-set-up-your-projects-sass-entry-point).
+
+All USWDS settings are defined in the `uswds-core` module, which means the USWDS settings configuration rule is `@use "uswds-core" with ()`.
```

Comment:
Added these paragraphs because I didn't want to make the assumption that users know how `@use` works and where this configuration should live. It feels a bit clunky to me, so if you have any polish to rub on it, please let me know!

Reply:
I think this looks awesome!! The only part where I feel it's a bit clunky (I probably wouldn't have noticed if you hadn't said anything) is line 30, which could potentially move to the start of line 28? It's like a bit of context you may need before following the instructions.

This is so beautiful and clear I almost feel like I could do it!

Reply:
Thanks @bonnieAcameron! 

As far as moving line 30, I think it needs to stay in the current order. Here's why:
One other way to say `@use...with()` is `@use <url> with()`. So line 28 is explaining generally what the `@use <url> with()` rule is and provides links to more information about how that rule works (This was done because this technique is relatively new and can be confusing even to users who are familiar with Sass). Then line 30 gives the specific USWDS `<url>` ("uswds-core") that is needed to complete the rule. 

If you see another way of writing this, please let me know!